app/controllers/courses_controller.rb

```diff
@@ -31,4 +33,10 @@ def apply
   rescue JsonApiClient::Errors::NotFound
     render template: 'errors/not_found', status: :not_found, formats: [:html]
   end
+
+private
+
+  def course_cache_key
+    "ttapi-course-#{params[:course_code]}-#{params[:provider_code]}"
```

Comment:
Do we sanitise or check these params in any way before using them as cache keys? I wonder how vulnerable this would be to over-stuffing the cache?

Reply:
Good point! We don't currently, but in this case the cache key doesn't get stored unless it's a valid course.